ironfish-cli/src/commands/status.ts

```diff
@@ -18,15 +18,19 @@ export default class Status extends IronfishCommand {
       default: false,
       description: 'follow the status of the node live',
     }),
+    all: Flags.boolean({
+      default: false,
+      description: 'show all status information',
+    }),
   }
 
   async start(): Promise<void> {
     const { flags } = await this.parse(Status)
 
     if (!flags.follow) {
       const client = await this.sdk.connectRpc()
-      const response = await client.status()
-      this.log(renderStatus(response.content))
+      const response = await client.getNodeStatus()
+      this.log(renderStatus(flags, response.content))
```

Comment:
if you care about the all flag within the function, you can pass directly. Also allows for more descriptive naming of the var.

Reply:
Changed